Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG PYTHON_VERSION=3.10
+FROM python:${PYTHON_VERSION}-slim AS base
+LABEL maintainer="yaroslavbordovoy@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
Consider using a virtual environment for installing Python packages to avoid potential conflicts with system packages. This can be done by creating a virtual environment in the Dockerfile and activating it before running `pip install`.